manifests/params.pp

```diff
@@ -22,7 +22,10 @@
       $virtinst_package = 'virtinst'
       $radvd_package = 'radvd'
       $sysconfig = false
-      $deb_default = {}
+      $deb_default = $::service_provider ? {
+        'systemd' => { 'libvirtd_opts' => '' },  # no '-d', it confuses systemd
```

Comment:
@nelsonjr I have multiple Ubuntu 16.04 boxes with the Puppet/Facter that ship with that version of the OS, they all report service provider as debian:

`root@Ubuntu-1604-xenial-64-minimal /etc/puppet # facter -p service_provider
debian
`
My puppet environment is  Puppet 3.8.5, Facter 2.4.6, Stdlib 4.12, masterless. What's yours?